### ISSUE TITLE
Radio button problem

### ISSUE THREAD

**Issue by martintorok** [erikras/redux-form-material-ui#41]
Hello, I am having trouble to set value to radio button programmatically.

I am able to dispatch a change action for radio buttons the same way as  nickbarry in his post here https://github.com/erikras/redux-form/issues/369

I am also able to see that Redux store value has been succesfully changed, in redux dev tools in chrome.

My problem is that Radio Button Group component on my page is not reflecting this change.
I tried similar thing for a TextBox and it worked as I expected.

I am still quite fresh to React and Redux, but I think following could be related problem - when you call reset on a form, Radio buttons are not resetted for material design wrappers but for normal Radio buttons reset works. Here is example that does not work for radio buttons. http://erikras.github.io/redux-form-material-ui/

Could the reason be that mapping for onchange is missing in the wrapper, or something?

Thanks!

**Comment by Philin-Anton** [comment]
+1

**Comment by Philipp91** [comment]
Same issue. This might be fixed with https://github.com/erikras/redux-form-material-ui/commit/e709604b5dba0b6d971e2071f93c3815ef8d4db7 (unreleased)

**Comment by tobiasrolsen** [comment]
+1

**Comment by vishnusajev** [comment]
This issue is still happening to me, did it get fixed?

**Comment by vishnusajev** [comment]
This problem is still happening to me, I tried to dispatch a change event to change the value of the RadioButtonGroup field and it is not reflecting until I dispatch the same even again.

**Comment by Philipp91** [comment]
I use the following component as a workaround (just put in a file like `ReduxFormRadioButtonGroup.js`):

```
"use strict";

import {RadioButtonGroup} from "material-ui/RadioButton";
import createComponent from "redux-form-material-ui/lib/createComponent";
import mapError from "redux-form-material-ui/lib/mapError";

const mapValueToValueSelected = ({input: {...inputProps}, ...props}, errorProp) =>
    mapError({...props, input: {...inputProps, valueSelected: inputProps.value}}, errorProp);

export default createComponent(RadioButtonGroup, mapValueToValueSelected);
```

The problem has indeed not been fixed by e709604, quite contrary, I now additionally get a `Warning: Failed prop type: The prop 'name' is marked as required in 'RadioButtonGroup', but its value is 'undefined'.` that I haven't debugged further.

**Comment by mihirsoni** [comment]
@Philipp91 Which MUI version are you using ?

**Comment by Philipp91** [comment]
I have 0.18.6.

**Comment by mihirsoni** [comment]
This has been Published in [`v4.1.0`](https://github.com/erikras/redux-form-material-ui/releases/tag/v4.1.0).

Closing this for now. If issue still persist feel free to reopen.

**Comment by Philipp91** [comment]
Confirmed, thanks!
